dbt_subprojects/daily_spellbook/models/_projects/gmx/event/avalanche_c/gmx_v2_avalanche_c_order_executed.sql

```diff
@@ -2,7 +2,9 @@
   config(
     schema = 'gmx_v2_avalanche_c',
     alias = 'order_executed',
-    materialized = 'table'
+    materialized = 'incremental',
+    unique_key = ['tx_hash', 'index'],
```

Comment:
By the way, we need to apply uinqueness tests to `unique_key` in the schema for every incremental spell:
i.e.
```
    data_tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - tx_hash
            - index
```

Reply:
Okay. I will add it to schemas. Thank you!

Reply:
Done ✅